### DISCUSSION TITLE
site-log/

### DISCUSSION THREAD

**Discussion by giscus** [export-nan/nan-blogs#12] [General]
# site-log/

起源 站点技术选型 技术栈 markdown vuepress Github pase Github actions 思路：博客文档部分采用 markdown 进行编写，将github仓库作为我们的文章数据源，使用 vuepress 对文章进行静态文件编译，采用 vuepress-theme-hope 进行主题开发，借助 Github actions 进行自动化部署。

http://localhost:8080/site-log/

**Comment by open-nan** [comment]
你好，欢迎你来到我的个人博客，如果我的笔记有帮到你不妨到github上给我点一个小星星吧
